File: Assignment_18/Program2.c
/*
Accept N numer from the user and display such element which are divided by 5 
*/

#include<stdio.h>
#include<stdlib.h>

void Display(int *Arr,int iLength)
{
    int i = 0;

    for(i = 0 ; i < iLength ; i++)
    {
        if(Arr[i] % 5 == 0)
        {
            printf("%d\n",Arr[i]);
        }
    }
}

int main()
{
    int iSize = 0 ,  iCnt = 0;

    int *ptr = NULL;

    printf("Enter the numbers of elements :\n");

    scanf("%d",&iSize);

    ptr = (int*)malloc(iSize * sizeof(int));

    if(ptr == NULL)
    {
        printf("Unable to allocate the memory\n");
        return -1;
    }

    printf("Enter the %d elements in the array\n",iSize);

    for(iCnt = 0; iCnt < iSize ; iCnt++)
    {
        printf("Enter %d element : ",iCnt+1);
        scanf("%d",&ptr[iCnt]);
    }

    Display(ptr,iSize);

    free(ptr);
    
    return 0;
}

File: Assignment_21/Program2.java
//Accept N Numbers from the user and return smallest element from the array
import java.util.*;

class ArrayX
{
    public int Arr[];

    public ArrayX(int iSize)
    {
        Arr = new int[iSize];
    }

    public void Accept(Scanner sobj)
    {
        int iCnt = 0;

        System.out.println("Enter "+Arr.length+" Elements");

        for(iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            Arr[iCnt] = sobj.nextInt();
        }
    }


    public int DisplayLargest()
    {
        int iCnt = 0;

        int iSmall = Arr[0];

        for(iCnt = 0 ; iCnt < Arr.length; iCnt++)
        {
            if(Arr[iCnt] < iSmall)
            {
                iSmall = Arr[iCnt];
            }
        }

        return iSmall;
    }


}

class Program2 
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the size of array");

        int iSize = sobj.nextInt();

        ArrayX aobj = new ArrayX(iSize);

        aobj.Accept(sobj);

        int iRet = aobj.DisplayLargest();

        System.out.println("The smallest element int the  array is : "+iRet);

        sobj.close();
    }
}
